backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/cfg/ElasticsearchBackendSettings.java

```diff
@@ -341,6 +342,22 @@ private ElasticsearchBackendSettings() {
 	 */
 	public static final String SCROLL_TIMEOUT = "scroll_timeout";
 
+	/**
+	 * Property for specifying the duration of time in milliseconds which the connection to perform requests
+	 * to Elasticsearch cluster can be safely kept idle.
+	 * <p>
+	 * Expects a positive Integer value of milliseconds, such as 60000,
+	 * or a String that can be parsed into such Integer value.
+	 * <p>
+	 * If received response from an Elasticsearch cluster contains a {@code Keep-Alive} header - then its timeout value
+	 * will be honored if it's less than the one configured with this property.
+	 * <p>
+	 * If the value is not set then the default {@link ConnectionKeepAliveStrategy keep alive strategy} will be used.
+	 * <p>
+	 * Value is not set by default.
```

Comment:
This is a bit confusing, especially the default since it requires knowning what `ConnectionKeepAliveStrategy` does exactly. Let's sum it up?

```suggestion
	 * If the response from an Elasticsearch cluster contains a {@code Keep-Alive} header,
	 * then the effective max idle time will be whichever is lower:
	 * the duration from the {@code Keep-Alive} header or the value of this property (if set).
	 * <p>
	 * If this property is not set, only the {@code Keep-Alive} header is considered,
	 * and if it's absent, idle connections will be kept forever.
```